kafka_utils/kafka_consumer_manager/commands/offset_manager.py

```diff
@@ -144,14 +144,23 @@ def get_topics_for_group_from_kafka(
         groupid,
     ):
         kafka_group_reader = KafkaGroupReader(cluster_config)
-        group_topics = kafka_group_reader.read_group(groupid)
-        if len(group_topics) == 0:
+        try:
+            group_topics = kafka_group_reader.read_group(groupid)
+        except KeyError:
             print(
-                "Error: Either Consumer Group {groupid} has never committed offsets, or the group does not exist.".format(
+                "Error: Consumer Group {groupid} does not exist.".format(
                     groupid=groupid,
                 ),
                 file=sys.stderr,
             )
+            group_topics = []
+        if len(group_topics) == 0:
+            print(
```

Comment:
Wonder if this print can be put in a different function, that would make the function itself more reusable. Are there times we call this function when we are not committing offsets?

I think in general the functions in `offset_manager` were structured a little weird. There's a tight coupling of the print messages and the behavior

Reply:
If I understand correctly it's always called when committing offsets.